controllers/validation/accountpool_validation_controller.go

```diff
@@ -0,0 +1,189 @@
+		for regionName, serviceQuota := range poolData.RegionedServicequotas {
+			var parsedServiceQuotas = make(awsv1alpha1.AccountServiceQuota)
+			for quotaCode, quotaValue := range serviceQuota {
+				qv, _ := strconv.Atoi(quotaValue)
+				parsedServiceQuotas[awsv1alpha1.SupportedServiceQuotas(quotaCode)] = &awsv1alpha1.ServiceQuotaStatus{
+					Value: qv,
+				}
+			}
+			parsedRegionalServiceQuotas[regionName] = parsedServiceQuotas
+		}
+	}
+
+	return parsedRegionalServiceQuotas, nil
+}
+
+func (r *AccountPoolValidationReconciler) checkAccountServiceQuota(reqLogger logr.Logger, accountPoolName string, parsedRegionalServiceQuotas awsv1alpha1.RegionalServiceQuotas) (ctrl.Result, error) {
+
+	accountList, err := r.getAccountPoolAccounts(accountPoolName)
+	if err != nil {
+		reqLogger.Error(err, "Failed to get AccountPool accounts")
+		return reconcile.Result{}, err
+	}
+
+	for _, account := range accountList {
+		if reflect.DeepEqual(account.Spec.RegionalServiceQuotas, parsedRegionalServiceQuotas) {
+			//if account.Spec.RegionalServiceQuotas != parsedRegionalServiceQuotas {
+		} else {
+			account.Spec.RegionalServiceQuotas = parsedRegionalServiceQuotas
+			account.Status.RegionalServiceQuotas = make(awsv1alpha1.RegionalServiceQuotas)
+		}
+
+	}
+
+	return reconcile.Result{}, nil
+}
+
+func (r *AccountPoolValidationReconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
+	reqLogger := log.WithValues("Controller", controllerName, "Request.Namespace", request.Namespace, "Request.Name", request.Name)
+
+	// Fetch the AccountPool instance
+	currentAccountPool := &awsv1alpha1.AccountPool{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: request.Name, Namespace: awsv1alpha1.AccountCrNamespace}, currentAccountPool)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			// Request object not found, could have been deleted after reconcile request.
+			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
+			// Return and don't requeue
+			return reconcile.Result{}, nil
+		}
+		// Error reading the object - requeue the request.
+		return reconcile.Result{}, err
+	}
+
+	// check if accountpool has servicequota defined in configmap
+	reginalServiceQuotas, err := r.checkAccountPoolServiceQuota(reqLogger, currentAccountPool.Name)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	_, err = r.checkAccountServiceQuota(reqLogger, currentAccountPool.Name, reginalServiceQuotas)
+
+	return utils.DoNotRequeue()
+}
+
+func (r *AccountPoolValidationReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	r.awsClientBuilder = &awsclient.Builder{}
+	maxReconciles, err := utils.GetControllerMaxReconciles(controllerName)
+	if err != nil {
+		log.Error(err, "missing max reconciles for controller", "controller", controllerName)
+	}
+
+	rwm := utils.NewReconcilerWithMetrics(r, accountpoolcontrollerName)
```

Comment:
I think you can replace the string object accountpoolcontrollerName with just the string `accountpoolvalidation`